.github/workflows/ci.yml

```
@@ -53,8 +53,8 @@ jobs:
       fail-fast: false
       matrix:
         go:
-          - 1.18
           - 1.19
+          - 1.20
```

YAML :man_shrugging:

```suggestion
          - "1.20"
```

---

🤦